docs/reference/settings/security-settings.asciidoc

```diff
@@ -1636,19 +1636,10 @@ at the OpenID Connect Provider.
 `rp.client_auth_method` {ess-icon}::
 (<<static-cluster-setting, Static>>)
 The client authentication method used by {es} to authenticate
-to the OpenID Connect Provider. Can be `client_secret_basic`, `client_secret_post`,
-or `client_secret_jwt`. Defaults to `client_secret_basic`.
+to the OpenID Connect Provider. Can be `client_secret_basic` or 
+`client_secret_post`. Defaults to `client_secret_basic`.
```

Comment:
JWT references should not be removed from OIDC realm. OIDC realm supports JWTs too.

Reply:
Thanks for the clarification -- I reverted this change.